.github/workflows/pre-submit.yaml

```diff
@@ -1,5 +1,6 @@
-name: Tests and build open and merged PRs
-on:
+# Understanding the workflow file - https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions#understanding-the-workflow-file
+name: Pre Submit # workflow name
+on: # on events
   push:
     branches:
       - master
```

Comment:
while we are here, we can remove master

Reply:
I want to do it in #23 . But the e2e tests is flaky  :(